examples/spread/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+use leptos::*;
+
+/// Demonstrates how attributes and event handlers can be spread onto elements.
+#[component]
+pub fn SpreadingExample() -> impl IntoView {
+    fn alert(msg: impl AsRef<str>) {
+        let _ = window().alert_with_message(msg.as_ref());
+    }
+
+    let attrs_only: Vec<(&'static str, Attribute)> = vec![
+        ("data-foo", Attribute::String(Oco::Borrowed("42"))),
```

Comment:
Is there a less explicit way of constructing `Attribute`? This is pretty imposing for an example.

Reply:
.into_attribute() exists

Reply:
Switched to `.into_attribute()` calls. Much more readable, and the explicitness is not needed here.